app/views/application/scheme/chooseyourschemes.scala.html

```diff
@@ -49,7 +55,7 @@ <h1 class="heading-xlarge" xmlns="http://www.w3.org/1999/html">Choose your prefe
                                 <label for="scheme-@scheme.id" class="block-label block-label-slim">
                                     <input type="checkbox" id="scheme-@scheme.id" data-schemename value="@scheme.id">@scheme.name</input>
                                 </label>
-                                <span class="alwayshidden" data-scheme-req>@schemeRequirements(scheme)</span>
+                                <span class="alwayshidden" @if(isSpecialScheme(scheme)){data-spec-scheme} else {data-scheme-req}>@schemeRequirements(scheme)</span>
```

Comment:
we have a java script which applies red background to the required qualifications if the selected scheme is special (means requires A-Level/ Stem level). This is how it is called in css as well as javascript.